test/CadlRanchProjects/inheritance/inheritance.tsp

```diff
@@ -64,6 +66,39 @@ model GoblinShark extends Shark {
   sharktype: "goblin";
 }*/
 
+@doc("extensible enum type for discriminator")
+enum DogKind {
+    Golden: "golden",
+}
```

Comment:
Add tests in cadl-ranch.

Reply:
> Add tests in cadl-ranch.

It won't work because the double discriminator is still there.

Reply:
@pshao25  @archerzz ,  as this typespec file was put under cadlRanchProjects folder,  could you explain why we need this additional file?

Reply:
My preference is that we check in the codes on our side. Then we deal with the cases in `cadl-ranch`. I don't want to waste time in the reviewing and arguing about the cases of inheritance which we know there is controversy.

Reply:
I believe every PR will face the same situation you are describing. If we do "first autorest.csharp then cadl-ranch" for every PR, there would be even more time waste on transferring.